pages/question.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from "react";
+import Quiz from "../components/quiz/Quiz";
+import type { Question } from "../server/db";
+function Questions() {
+  const [choice, setChoice] = useState<Question>(null);
+
+  useEffect(() => {
+    fetch("/api/quizanswers")
+      .then((response) => response.json())
+      .then((response) => setChoice(response[0]));
```

Comment:
```suggestion
      .then((result) => setChoice(result[0]));
```
or
```suggestion
      .then((quizAnswers) => setChoice(quizAnswers[0]));
```

Reply:
Why is it required to read only the first answer?

Reply:
> Why is it required to read only the first answer?

don't understand what you meant, "read-only the first answer"?

Reply:
Because you access `[0]`. Why do you send an array from your api if you only need the first entry?